lang/java/reef-tests/src/test/java/org/apache/reef/tests/evaluatorfailure/EvaluatorFailureDuringAlarmDriver.java

```diff
@@ -70,6 +71,12 @@ public void onNext(final FailedEvaluator failedEvaluator) {
     }
   }
 
+  final class ActiveContextHandler implements EventHandler<ActiveContext> {
```

Comment:
Actually, the test on Windows fails after implementing OSUtils fully. The reason is `DefaultContextActiveHandler` really works immediately as I wrote in JIRA issue.
```
Add No-Op `ActiveContextHandler` to EvaluatorFailureDuringAlarmDriver. `DefaultContextActiveHandler` suicides by closing itself.
```